docs/operate/get-started/other-hardware/create-module/_index.md

```diff
@@ -0,0 +1,1102 @@
+---
+title: "Create a module"
```

Comment:
also previous page should have a next button linking to this, meta.json should have next link to update and manage and please check for any other dangling ends. The idea is that this is a path through the docs you can click.

Reply:
Agree with button to this create page, but for meta.json I should actually remove previous and not have any forward/back buttons right, since it's just a reference page and not part of flow per se

Reply:
I get that it doesn't feel like a next step but wouldn't be too strict about that. If we interleave the reference pages then they become part of the flow.
So I think meta.json should continue to update and manage modules and I think module configuration can continue to lifecycle of a module.